test/run-drun/ok/migrate-upgrade-instr-stable-mem.drun.ok

```diff
@@ -0,0 +1,7 @@
+ingress Completed: Reply: 0x4449444c016c01b3c4b1f204680100010a00000000000000000101
+debug.print: Stable memory size: 10
+ingress Completed: Reply: 0x4449444c0000
+debug.print: Stable memory size: 20
+ingress Completed: Reply: 0x4449444c0000
+debug.print: Upgrade instructions: 18_446_744_073_709_551_615
```

Comment:
Why is the instruction count so large? Is it really doing that much work?

Reply:
It is a sentinel (u64::MAX) if the upgrade instructions are unknown when upgrading from an old canister that did not record the pre-upgrade costs.

Reply:
(sorry, just saw your comment below :) )